app/src/org/commcare/dalvik/dialogs/HorizontalPaneledChoiceDialog.java

```diff
@@ -0,0 +1,38 @@
+package org.commcare.dalvik.dialogs;
+
+import android.content.Context;
+import android.view.LayoutInflater;
+import android.widget.Button;
+
+import org.commcare.dalvik.R;
+
+/**
+ * A specialized version of PaneledChoiceDialog that shows exactly 3 choice items (rather than N
+ * items) in a horizontal (rather than vertical) orientation
+ *
+ * @author amstone
+ */
+public class HorizontalPaneledChoiceDialog extends PaneledChoiceDialog {
+
+    public HorizontalPaneledChoiceDialog(Context context, String title) {
+        super(context, title);
+        view = LayoutInflater.from(context).inflate(R.layout.choice_dialog_three_panel, null);
```

Comment:
@phillipm Seems pretty clean to me honestly, will go with that for now. Thanks for all the feedback.